js/index.js

```diff
@@ -0,0 +1,32 @@
+$(document).ready(function() {
+
+$('#submit-btn').click(submitOnClick);
+//  We are now in the button.submit `click` event listener
+  function submitOnClick(event) {
+    event.preventDefault();
+
+    var cityName = $('#city-type').val().trim();
+    cityName = cityName.toLowerCase();
+
+    if (cityName === 'new york city' || cityName === 'new york' || cityName ===  'nyc') {
+      $('body').addClass('nyc').removeClass('sf','la','austin','sydney');
+      $('#city-type').val('');
```

Comment:
You could have put this once on line 10 (declaring it just that one time) instead of repeating for every call.